ui/templates/welcome.html

```diff
@@ -29,3 +30,14 @@
         </div>
     {% endif %}
 {% endblock %}
+{% block postcontent %}
+{{ block.super }}
+{% compress js %}
+  <script type="text/javascript"
```

Comment:
Agreed @noisecapella. We noticed that this also occurs on the create repository page, and since these two are required for any page we create, I think it makes sense to put them in base so this works on all pages without duplicating code everywhere.